src/Metadata/Resource/Factory/ResourceMetadataFactoryTrait.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace ApiPlatform\Core\Metadata\Resource\Factory;
+
+use ApiPlatform\Core\Exception\ResourceClassNotFoundException;
+use ApiPlatform\Core\Metadata\Resource\Operation;
```

Comment:
+1 and there is a call:

```
                $paginationMetadata = new PaginationMetadata(
                    $paginationAnnotation->enabled,
                    (float) $paginationAnnotation->itemsPerPage,
                    $paginationAnnotation->clientControlEnabled
                );

```

I was searching for the class but could not find it. What should I do about them?